dart/lib/src/sentry_client.dart

```diff
@@ -80,9 +85,10 @@ class SentryClient {
       return _sentryId;
     }
 
-    if (_options.beforeSend != null) {
+    final beforeSend = _options.beforeSend;
```

Comment:
This way the dart compiler can infer that beforeSend is not null and the `!` isn't needed anymore.